bundles/EcommerceFrameworkBundle/IndexService/Worker/ElasticSearch/AbstractElasticSearch.php

```diff
@@ -866,7 +879,11 @@ protected function checkIndexLock(bool $throwException = true) : bool {
             if ($throwException) {
                 throw new \Exception($errorMessage);
             } else {
-                Logger::warning($errorMessage);
+                //only write log message once a minute to not spam up log file when running update index
+                if($this->lastLockLogTimestamp > time() + 60) {
```

Comment:
The first time ``$this->lastLockLogTimestamp`` is null, so probably Travis will complain.
Either cast it to an integer or initialize ``lastLockLogTimestamp`` with 0 instead of null.

Reply:
done